test/e2e/commands/mirror_node.test.mjs

```diff
@@ -152,6 +153,8 @@ describe('MirrorNodeCommand', () => {
     } catch (e) {
       mirrorNodeCmd.logger.showUserError(e)
       expect(e).toBeNull()
+    } finally {
+      await accountManager.close()
```

Comment:
do we need accountManager.close() in both placees?

Reply:
done

Reply:
removed